app/models/links.rb

```diff
@@ -0,0 +1,57 @@
+class Links
```

Comment:
This is our new CS Links API interface. It also supports the 'My Campus' page links that we plan on migrating to the CS Link API via the `#campus_links` method.